### DISCUSSION TITLE
Feature Request: Add documentation for the Excel extension

### DISCUSSION THREAD

**Discussion by frafra** [duckdb/duckdb#6340] [Ideas]
Hi!
DuckDB has an extension called `Excel`, but it is barely mentioned in the documentation, under `All available extensions`. It would be nice to read more about it and see an example.

**Comment by Mytherin** [comment]
The `excel` extension does not contain much at the moment. It was created by an outside contributor and it only includes Excel's numeric format functions ported from LibreOffice. See [this test file for the full extent of what the excel extension currently enables](https://github.com/duckdb/duckdb/blob/master/test/sql/excel/test_excel_numformat.test).
